src/dut.rs

```diff
@@ -698,7 +699,7 @@ impl SshInfo {
                     (child, port)
                 } else {
                     eprintln!("Failed to establish ssh port forwarding");
-                    return;
+                    exit(1);
```

Comment:
Can't we return some error instead of forcibly exit? Or, at least it should be commented at the head of the function.

Reply:
Updated to return error when the first connection attempt is failed.